integrationtest/wildfly/pom.xml

```diff
@@ -374,6 +374,39 @@
                             </resources>
                         </configuration>
                     </execution>
+                    <!-- Temporary workaround for HHH-11394 -->
```

Comment:
I didn't mean it as a strict rule, more like something to keep in mind.

New rule:
 - there never are strict rules.

I typically see value in keeping `git bisect` working fine; this is more valuable in maintenance branches as the likelyhood for it to be broken anyway during a large feature sprint is high.

There's also value in keeping changes indipendent. In this case I felt keeping each step of the ORM upgrade independent was making it clearer both for you to review, and for future reference.

If you feel strongly about it I can squash them all; what do you prefer?